compiler/rustc_target/src/spec/mod.rs

```diff
@@ -1316,6 +1316,32 @@ bitflags::bitflags! {
 rustc_data_structures::external_bitflags_debug! { SanitizerSet }
 
 impl SanitizerSet {
+    const MUTUALLY_EXCLUSIVE: &'static [(SanitizerSet, SanitizerSet)] = &[
```

Comment:
Can you add the link @ilovepi posted as a cross reference here?

Reply:
Done